roles/confluent.kafka_connect_replicator/tasks/main.yml

```diff
@@ -0,0 +1,307 @@
+    group: "{{kafka_connect_replicator_group}}"
+    hostnames: "{{ [inventory_hostname, ansible_fqdn, ansible_host, ansible_ssh_host] | unique }}"
+    ca_cert_path: "{{kafka_connect_replicator_consumer_ca_cert_path}}"
+    cert_path: "{{kafka_connect_replicator_consumer_cert_path}}"
+    key_path: "{{kafka_connect_replicator_consumer_key_path}}"
+    export_certs: "{{kafka_connect_replicator_consumer_export_certs}}"
+    ssl_ca_cert_file_path: "{{kafka_connect_replicator_consumer_ssl_ca_cert_path}}"
+    ssl_signed_cert_filepath: "{{kafka_connect_replicator_consumer_ssl_cert_path}}"
+    ssl_key_filepath: "{{kafka_connect_replicator_consumer_ssl_key_path}}"
+    ssl_key_password: "{{kafka_connect_replicator_consumer_ssl_key_password}}"
+    ssl_truststore_file_path: "{{kafka_connect_replicator_consumer_ssl_truststore_file_path}}"
+    ssl_keystore_file_path: "{{kafka_connect_replicator_consumer_ssl_keystore_file_path}}"
+  when: kafka_connect_replicator_consumer_ssl_enabled|bool or kafka_connect_replicator_consumer_ssl_provided_keystore_and_truststore|bool
+
+- name: Configure TLS for Kafka Connect Replicator Producer
+  include_role:
+    name: confluent.ssl
+  vars:
+    truststore_storepass: "{{kafka_connect_replicator_producer_truststore_storepass}}"
+    truststore_path: "{{kafka_connect_replicator_producer_truststore_path}}"
+    keystore_path: "{{kafka_connect_replicator_producer_keystore_path}}"
+    keystore_storepass: "{{kafka_connect_replicator_producer_keystore_storepass}}"
+    keystore_keypass: "{{kafka_connect_replicator_producer)keystore_keypass}}"
+    service_name: kafka-connect-replicator
+    user: "{{kafka_connect_replicator_user}}"
+    group: "{{kafka_connect_replicator_group}}"
+    hostnames: "{{ [inventory_hostname, ansible_fqdn, ansible_host, ansible_ssh_host] | unique }}"
+    ca_cert_path: "{{kafka_connect_replicator_producer_ca_cert_path}}"
+    cert_path: "{{kafka_connect_replicator_producer_cert_path}}"
+    key_path: "{{kafka_connect_replicator_producer_key_path}}"
+    export_certs: "{{kafka_connect_replicator_producer_export_certs}}"
+    ssl_ca_cert_file_path: "{{kafka_connect_replicator_producer_ssl_ca_cert_path}}"
+    ssl_signed_cert_filepath: "{{kafka_connect_replicator_producer_ssl_cert_path}}"
+    ssl_key_filepath: "{{kafka_connect_replicator_producer_ssl_key_path}}"
+    ssl_key_password: "{{kafka_connect_replicator_producer_ssl_key_password}}"
+    ssl_truststore_file_path: "{{kafka_connect_replicator_producer_ssl_truststore_file_path}}"
+    ssl_keystore_file_path: "{{kafka_connect_replicator_producer_ssl_keystore_file_path}}"
+  when: kafka_connect_replicator_producer_ssl_enabled|bool or kafka_connect_replicator_producer_ssl_provided_keystore_and_truststore|bool
+
+- name: Configure TLS for Kafka Connect Replicator Monitoring Interceptors
+  include_role:
+    name: confluent.ssl
+  vars:
+    truststore_storepass: "{{kafka_connect_replicator_monitoring_interceptor_truststore_storepass}}"
```

Comment:
add additional role variables for
```
ssl_custom_certs:
provided_keystore_and_truststore:
```

and then you wont need to modify the ssl role source code